docs-redirect.html

```diff
@@ -1,10 +1,16 @@
 <script>
   const path = window.location.pathname;
-  if (
+
+  if (path.includes("/overview/history.html")) {
```

Comment:
making sure we want to avoid a situation where someone wants to look at an old history page. 

```suggestion
  if (
    (
      path.startsWith("/en/latest") || 
      path.startsWith("/en/stable")
    ) && 
    path.includes("overview/history.html")
  ) {
```